src/commands/Information/channelinfo.ts

```diff
@@ -16,6 +16,7 @@ export default class extends Command {
 			clearance: 50,
 			maxArgs: 1,
 			clientPerms: ["EMBED_LINKS"],
+			argList: ["channel:Channel"],
```

Comment:
```suggestion
			argList: ["channel:ChannelText"],
```

Reply:
No beacuse you can channelinfo all channels in a server

Reply:
got it, sorry